jwst/tests_nightly/general/miri/test_miri_steps.py

```diff
@@ -58,7 +58,7 @@ class TestMIRISteps(BaseJWSTTestSteps):
                       step_pars=dict(),
                       output_truth='jw00001001001_01101_00001_MIRIMAGE_dark_current.fits',
                       output_hdus=[],
-                      id='test_dark_current_miri'
+                      id='dark_current_miri'
```

Comment:
Could shorten these even further, if necessary, to just "dark_miri"

Reply:
True!  Though the step is called `jwst.dark_current.DarkCurrentStep`.  I think the above is short enough to not run into any of these problems anymore.